README.md

```diff
@@ -1 +1,80 @@
-# achinn
\ No newline at end of file
+# achinn
+
+This is a living document about me.
+
+It provides a way for others to quickly get up to speed with how I work.
+
+It provides a way for me to build my own self awareness.
+
+
+## Public endpoints
+
+### `/slack`
+
+If you need something immediate from me, this or [`/ad-hoc-irl`](#ad-hoc-irl) are the endpoints to use. For the most part I respond quickly but [there are](#context-switching) [instances](#meetings) where I may not. In those cases I will leave your message unread and circle back later. If I'm offline, however, all bets are off.
+
+### `/email`
+
+I treat my inbox with less urgency and tend to poll it when I'm in between tasks or looking for the next task to tackle (e.g. PR updates).
+
+### `/meetings`
+
+I like to know what a meeting is about before accepting it. If there's no description on a meeting request, don't be surprised if I ask for more details after receiving it.
+
+If I'm in a meeting I will try my absolute hardest to give you 100% of my attention. This is why I prefer not to bring my laptop to most meetings. My phone will also stay in my pocket unless an emergency call comes up. *Please call me out if you see me breaking either of these rules of mine.*
+
+Wrapping up the material before the allotted time is a _good thing_. It means we can all have some time back.
+
+### `/1-to-1s`
+
+This time is yours to tailor. The frequency and length of these meetings is up to you.
+
+Let's talk about anything that you feel would help you get better at what you do and anything that's getting in your way. If you can, spend some time before each meeting thinking about what you want to cover. Remember, this is about you -- don't squander it.
+
+### `/pair-programming`
+
+This is one of my favourites. Working through a problem with someone else is exciting. By the end of our time together we've created an artifact _and_ we most likely learned some things from each other.
```

Comment:
Are there any particular ways you like to pair? Or expectations of your pair?

Reply:
```suggestion
This is one of my favourites. Working through a problem with someone else is exciting. By the end of our time together we've created an artifact _and_ we most likely learned some things from each other.

Typically I like to allot two hours to our pairing session. I find that's a good amount of time to spend on a problem as it also allows for discussion and/or whiteboarding (did you think we'd just be heads down coding?). As for _what_ we work on, it doesn't matter to me; we can tackle something either one of us has been assigned.
```